scripts/import_mongodb.sh

```
@@ -72,14 +92,11 @@ import_file() {
         fi
         
-        # Use mongoimport to import the file
-        docker run --rm -v "$JSON_DIR:/data" mongo:latest \
-            mongoimport --uri="$MONGO_URI" \
-            --db="$DB_NAME" \
-            --collection="$COLLECTION_NAME" \
-            --file="/data/$(basename "$FILE")" \
-            $DROP_OPTION \
-            $TYPE_OPTS
+        # Common mongoimport arguments
+        local IMPORT_ARGS=(--uri="$MONGO_URI" --db="$DB_NAME" --collection="$COLLECTION_NAME" $DROP_OPTION $TYPE_OPTS)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)